aws-mssql.yml

```diff
@@ -11,8 +11,33 @@ support_url: https://aws.amazon.com/sql/
 tags: [aws, mssql, beta]
 plan_updateable: true
 provision:
-  plan_inputs: [] # Expected: mssql_version, storage_gb
+  plan_inputs: []
   user_inputs:
+  - field_name: engine
+    required: true
+    type: string
+    details: The edition for the MSSQL instance.
+    default: sqlserver-ee
```

Comment:
Can we make this nullable? If users want to use the same one always they can configure it in their plans. We don't usually set versions or alike in our plans. It is also flagged as None for default in our kickoff board.

Reply:
Sure. Good point. Thanks for noticing.
Done.